### ISSUE TITLE
Copy-paste issues on CodinGame

### ISSUE THREAD

**Issue by 2StepsFr0mHell** [vadim-job-hg/Codingame#3]
Hello,

Your repository is a gold mine for players on CodinGame. However, most of the time, people just copy-paste your AI bots and submit them without bothering much or learning from it. So globally, your repo is harming the competition on CodinGame.

Would you consider closing it?

Thibaud from CodinGame

**Comment by maxim25111** [comment]
Yes, I agree.. Actually, I make my one code in c++, and my code is 255 lines long, and is able to win with bot written of 750 line long code... Its strange, but its true. Importing is logic.
Unfortunately, there are now to much peoples with copy of that code, and they stuck middle of list of coders, now is even hard to break that 500 place. Reason is that not all maps are this same, and sometimes bad starting place can cause to user fail in game, and other time, winn. But if I need play with 50 or even 100 users using copy paste code, I will never break that 500 level. 
I play in CodiGame from 2 Years, and many time I faced that situation where copy paste code someone else break funn, and hurt real skills of peoples who try code 5 days, and at the end, they give up because they could not brake that magically level of 100 peoples using copy paste code.

Please close this project, Your code I could use as learning purpose, but I do it because I want learn. But really.... If You want help peoples who don't want think, and only go to easy one, and cheat looking for job, please, remove it.
